p2p/src/authenticated/discovery/actors/tracker/actor.rs

```diff
@@ -854,6 +860,35 @@ mod tests {
         });
     }
 
+    #[test]
+    fn test_listenable() {
+        let executor = deterministic::Runner::default();
+        executor.start(|context| async move {
+            let (peer_signer, peer_pk) = new_signer_and_pk(0);
+            let (_peer_signer2, peer_pk2) = new_signer_and_pk(1);
+            let (_peer_signer3, peer_pk3) = new_signer_and_pk(2);
+            let cfg_initial = default_test_config(peer_signer, Vec::new());
+            let TestHarness {
+                mut mailbox,
+                mut oracle,
+                ..
+            } = setup_actor(context.clone(), cfg_initial);
+
+            assert!(!mailbox.listenable(peer_pk.clone()).await);
+            assert!(!mailbox.listenable(peer_pk2.clone()).await);
+            assert!(!mailbox.listenable(peer_pk3.clone()).await);
+
+            oracle
+                .register(0, vec![peer_pk.clone(), peer_pk2.clone()])
+                .await;
+            context.sleep(Duration::from_millis(10)).await;
+
+            assert!(!mailbox.listenable(peer_pk).await);
+            assert!(mailbox.listenable(peer_pk2).await);
```

Comment:
```suggestion
            // Not listenable because its myself
            assert!(!mailbox.listenable(peer_pk).await);
            // Listenable because it's registered
            assert!(mailbox.listenable(peer_pk2).await);
            // Not listenable because it's not registered
```